### PR TITLE
Add voice channel join/leave logging

### PR DESCRIPTION
**Pull request by ianagbip1oti**

### CODE REVIEW

File: src/main/kotlin/me/moeszyslak/logbot/listeners/VoiceListener.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import me.moeszyslak.logbot.dataclasses.Configuration
+import me.moeszyslak.logbot.embeds.createVoiceJoinEmbed
+import me.moeszyslak.logbot.embeds.createVoiceLeaveEmbed
+
+fun voiceListener(configuration: Configuration) = listeners {
+    on<VoiceStateUpdateEvent> {
+        val guildId = state.guildId ?: return@on
+        val guildConfig = configuration[guildId.longValue] ?: return@on
+
+        val channel = kord.getChannelOf<TextChannel>(guildConfig.historyChannel.toSnowflake())
```
Comment (ItsAzaria) [inline src/main/kotlin/me/moeszyslak/logbot/listeners/VoiceListener.kt]: I think this would be better in the log channel rather than history

**Review by ItsAzaria — CHANGES_REQUESTED** [review]
Other than the change about using the log channel rather than history the rest looks great

File: src/main/kotlin/me/moeszyslak/logbot/listeners/VoiceListener.kt (hunk 1 above)
Comment (ianagbip1oti) [reply src/main/kotlin/me/moeszyslak/logbot/listeners/VoiceListener.kt]: 👍 done

**Comment by ItsAzaria** [comment]
Cool, thanks for the PR

**Pull request merged**